twake/backend/node/src/core/platform/services/tracker/index.ts

```diff
@@ -5,6 +5,7 @@ import TrackerAPI from "./provider";
 import { localEventBus } from "../../framework/pubsub";
 import { IdentifyObjectType, TrackedEventType, TrackerConfiguration } from "./types";
 import { ResourceEventsPayload } from "../../../../utils/types";
+import { md5 } from "src/core/crypto";
```

Comment:
src should not appear in import

Reply:
Right, well the tests were going to tell me soon enough!